homeassistant/components/matter/light.py

```diff
@@ -78,6 +79,23 @@ def supports_brightness(self) -> bool:
             return False
         return ColorMode.BRIGHTNESS in self._attr_supported_color_modes
 
+    def supports_color_feature(self, feature: ColorControlCapabilities) -> bool:
```

Comment:
why are we going to call this every time ? The featureflags are static so just grab once at entity init is enough.